tools/accuracy_checker/accuracy_checker/adapters/README.md

```diff
@@ -153,6 +153,12 @@ AccuracyChecker supports following set of adapters:
   * `u_output` - U channel output layer.
   * `v_output` - V channel output layer.
   * `target_color` - taret color space for super resolution image - `bgr` and `rgb` are supported. (Optional, default `bgr`).
+* `super_resolution_gray` - converts output of super resolution model, which return output in GRAY format, to `SuperResolutionPrediction`.
```

Comment:
not sure that it is correct name, if model works with gray scale image, then it should return gray scale image, not BGR and not RGB

Reply:
Are you sure that you do every thing right?
according to description:

> Postprocessing involves converting the array of pixels into an image that is scaled to a higher resolution. The color channels (Cb, Cr) are also scaled to a higher resolution using bicubic interpolation. Then the color channels are combined and converted back to RGB format, producing the final output image.

It means that you use the first channel from YCbCr image as prediction and rescale other channels from the YCbCr image to new size